### DISCUSSION TITLE
Husky gives npm: command not found when running via VS Code

### DISCUSSION THREAD

**Discussion by ZennerBlade** [typicode/husky#1236] [Q&A]
Hi, 
Ever since I updated my Mac to Ventura, husky stopped working via VS Code commit/commit & sync.
It works perfectly when run via terminal.

My pre-commit file content

    #!/usr/bin/env sh
    . "$(dirname -- "$0")/_/husky.sh"

    npm run lint

Error

    > git -c user.useConfigOnly=true commit --quiet
    .husky/pre-commit: line 4: npm: command not found
    husky - pre-commit hook exited with code 127 (error)
    husky - command not found in PATH=/Library/Developer/CommandLineTools/usr/libexec/git-core:/usr/bin:/bin:/usr/sbin:/sbin

Any solution?

EDIT:
Got resolved automatically after restart.

**Comment by typicode** [comment]
What do you get if you run `echo $PATH` from the terminal?

**Comment by staszek998** [comment]
> EDIT:
> Got resolved automatically after restart.

It works! 🙂

**Comment by krutoo** [comment]
I have same issue, 

I read this article https://typicode.github.io/husky/how-to.html#node-version-managers-and-guis

And make `~/.config/husky/init.sh` with:

```
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

In my pre-commit hook i have `nvm use && npm run check && npm run lint`

But it is not working - still have:
```
Git: .husky/pre-commit: line 1: nvm: command not found
```

Also now event commit from zsh not working with same error:
```
.husky/pre-commit: line 1: nvm: command not found
husky - pre-commit script failed (code 127)
husky - command not found in PATH=...
```

How can i fix that?
